tests/test_updater.py

```diff
@@ -71,6 +71,16 @@ def test_update_with_additional_repos(self, run_command, fs):
             ['yum', 'update', '--disablerepo=*', '--enablerepo=' + ','.join(repos), '-y']
         )
 
+    def test_update_with_disabled_repo(self, run_command, fs):
+        run_command.return_value = {}
+
+        repos = list(DEFAULT_REPOS)
+        repos.append('ignored_repo')
+        update(mock.MagicMock(), {'repos': 'ignored_repo'})
+        run_command.assert_called_once_with(
+            ['yum', 'update', '--disablerepo=*', '--enablerepo=' + ','.join(DEFAULT_REPOS), '-y']
+        )
+
```

Comment:
Shouldn't there be a test for the new install function, too?

Reply:
Same I have a test function, not pushed yet.